src/app/app.scss

```diff
@@ -43,12 +44,29 @@ ion-alert.alert-top {
   font-size: 1.6rem;
 }
 
+// Prevent uppercase on buttons
+.button {
```

Comment:
This prevention is set on `variables.scss` as: ```$button-md-text-transform: none```

Reply:
Good catch, thanks! (cc @vanhue)